### ISSUE TITLE
[Bug]: Documentation for Rails is for Rails 6, but Rails is on Rails 7

### ISSUE THREAD

**Issue by pickhardt** [babel/babel#14776]
### 💻

- [ ] Would you like to work on a fix?

### How are you using Babel?

babel-loader (webpack)

### Input code

n/a just documentation needs updating

### Current and expected behavior

I was reading the Rails instructions at https://babeljs.io/setup#installation and it mentions Rails 6, but Rails is now on Rails 7, so it should probably get updated.

### Environment

n/a, just documentation

**Comment by nicolo-ribaudo** [comment]
Hi! No one in the current team uses Rails, so we would need help with this :sweat_smile: Would you like to open a pr in the `babel/website` repository updating it to "whatever needs to be done now"?

**Comment by Moshahrani** [comment]
> 

Hey Nicolo, is there anyway I can help contribute by updating the docs for this issue?? Thanks

**Comment by Sharanya98** [comment]
Is this issue open for contribution?

**Comment by nicolo-ribaudo** [comment]
Yes! No one in the team uses Rails, but please open a PR in the babel/website repository if the docs are still outdated :pray:
